### PR TITLE
Add asset manifest

### PR DESCRIPTION
**Pull request by brysonthor**
adds assets manifest and some gitignores for intellij

### CODE REVIEW

File: asset.manifest.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  "/components/birch-typeahead/birch-typeahead.css",
+  "/components/paper-item/paper-item-behavior.html",
+  "/components/paper-item/paper-item-shared-styles.html",
+  "/components/paper-styles/color.html",
+  "/components/paper-styles/default-theme.html",
+  "/components/paper-styles/typography.html"
+]
```
Comment (jpodwys) [inline asset.manifest.json]: Is this valid JSON? I think you need a key name and it needs to be either `bowerFiles` or `clientManifestFiles`.

Comment (jpodwys) [reply asset.manifest.json]: Here's the [Memories version of this file](https://github.com/fs-webdev/memories/blob/master/asset.manifest.json).

Comment (brysonthor) [reply asset.manifest.json]: I've followed this documentation let me know if you'd like the explicit version https://github.com/fs-webdev/frontier-build-tools/wiki/Bower-Asset-Manifest

Comment (brysonthor) [reply asset.manifest.json]: but as to my knowledge and testing this worked well

Comment (jpodwys) [reply asset.manifest.json]: Thanks, that was helpful.

**Pull request merged**